index.js

```diff
@@ -28,16 +26,18 @@ module.exports = {
     optimizeCache : require('./lib/optimizeCache'),
     packArray : require('./lib/packArray'),
     parseBinaryGltf : require('./lib/parseBinaryGltf'),
+    Pipeline : require('./lib/Pipeline'),
```

Comment:
It doesn't really matter in this case.  Since this won't be auto-sorted, whatever you guys want to do is fine.